frontend_v2/src/app/components/privacy-policy/privacy-policy.component.scss

```diff
@@ -19,8 +20,7 @@
 
     position: relative;
     position: -webkit-sticky;
-    position: sticky;
-    max-height: 65vh;
```

Comment:
max-height: 600px!important; is already present at line number 14. We do not need this definition again.